buffer/src/main/java/io/netty/buffer/AbstractByteBuf.java

```diff
@@ -1159,7 +1159,7 @@ protected final void checkDstIndex(int index, int length, int dstIndex, int dstC
         }
     }
 
-    private static boolean isInvalid(int index, int length, int capacity) {
+    static boolean isInvalid(int index, int length, int capacity) {
```

Comment:
sure we could... not sure if it's needed thought as I would like to keep it package private :)